### DISCUSSION TITLE
Launch blockers

### DISCUSSION THREAD

**Discussion by alecfwilson** [progressforum/ForumMagnum#14] [General]
Creating a list of remaining launch blockers, if the team has ideas on ETA, difficulty, or help requests for each of them, please comment.

- [X] Email backend, email confirmation, and email subscription support.
- [X] Set up production RSS feed
- [X] Integrate Maps provider and support community & events page
- [X] Update user profile page to remove non-applicable options
- [x] Create a Library page or remove it from the sidebar and footer
- [ ] Create basic tags, remove Wiki from sidebar
    - [X] Remove Wiki from sidebar 
- [ ] Add logos and favicon
- [ ] Update CSS
- [ ] Add about page
- [ ] Add contact us page

**Comment by alecfwilson** [comment]
Email is in progress

**Comment by alecfwilson** [comment]
Email confirmation is working

**Comment by sameersismail** [comment]
> Oauth support - are we going to support it? Which providers?

I don't think OAuth should be a blocking requirement for launch. (The LW people also mentioned it as a hassle, and, IIRC, mentioned it'd be preferable to start without it.)

> Set up production RSS feed

Took a brief look, and this doesn't seem to require anything special. (There's a cron, and the feed endpoint dynamically generates the XML file.)

E.g. hitting https://beta.progressforum.org/feed.xml?view=frontpage-rss&karmaThreshold=0 with `curl` returns what you'd expect.

---

Most of the other items look like an afternoon-ish of work each (with the small PF-specific modifications collapsed in one). The largest unknown to me is the UI design, do we have agreed upon mockups?

From an infrastructure perspective, the tasks to do are: (1) set up the production environment (AWS EB, MongoDB cluster, integrate third party services); and (2) monitoring/alerting (CloudWatch, Sentry, etc.).

I've created another column on [this project](https://github.com/Progress-Studies-Forum/ForumMagnum/projects/1) to separate non-blocking launch items. The  To Do column now being all blocking items.

**Comment by alecfwilson** [comment]
I am fine removing Oauth for launch

**Comment by sameersismail** [comment]
Removed the non-applicable options from the settings page.
